qml/SortTypeSelectionDialog.qml

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2019 Open Mobile Platform LLC
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2 only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import QtQuick 2.4
+import Sailfish.Silica 1.0
+import Sailfish.Office.Files 1.0
+
+Dialog {
```

Comment:
Please implement this like jolla-email has: pulldown menu having "Sort by: <type" and then resulting page allowing to just click on sorting label. We don't really need separate acceptance for this and the switches look strange too.

Reply:
Done